Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -3210,6 +3210,8 @@ void WebPage::updateFrameSize(WebCore::FrameIdentifier frameID, WebCore::IntSize
         return;
 
     frameView->resize(newSize);
+    // FIXME: This should be removed after rdar://122429810 is fixed.
+    frameView->setExposedContentRect(frameView->frameRect());
```

Comment:
Could we also add ASSERT(m_page->settings().siteIsolationEnabled()) to make it more clear that this won't affect anything when site isolation is off?